src/Kinto.elm

```diff
@@ -472,6 +509,62 @@ extractKintoError statusCode statusMsg body =
                 |> ServerError statusCode statusMsg
 
 
+withQueryParam : ( String, String ) -> Request a -> Request a
```

Comment:
This whole helper should probably be removed in favor of https://package.elm-lang.org/packages/elm/url/latest/Url-Builder#QueryParameter, what do you think?